library/core/src/convert/num.rs

```diff
@@ -163,7 +165,13 @@ impl_from!(u16 => f64, #[stable(feature = "lossless_float_conv", since = "1.6.0"
 impl_from!(u32 => f64, #[stable(feature = "lossless_float_conv", since = "1.6.0")]);
 
 // float -> float
+// todo: these are insta-stable regardless of the gate, right?
```

Comment:
Yes, these are insta-stable but they would just be included in the FCP for the types themselves.

Reply:
Thanks, I removed this note